test/include/RAJA_test-atomic-types.hpp

```diff
@@ -24,19 +21,10 @@
 using AtomicDataTypeList =
   camp::list< RAJA::Index_type,
               int,
-#if defined(RAJA_TEST_EXHAUSTIVE)
-              unsigned,
+              unsigned int,
```

Comment:
Do we still want to separate out some for the exhaustive tests?

Reply:
I'm torn. But thinking about it, int and double cover the majority of our use cases.

Reply:
OK, I'll leave int and double testing on for now and let the rest be enabled during exhaustive testing.